<?xml version="1.0"?>
<launch>

    <node pkg="articulation_structure" type="structure_learner_srv" name="structure_learner_srv" cwd="node" respawn="false" />

    <node pkg="cob_mmcontroller" type="cob_articulation_cartcollector.py" name="cob_articulation_cartcollector" cwd="node" output="screen">
        <param name="downsample" value="True" />
        <!--param name="sigma_orientation" value="0.5" /-->
    </node>

</launch>
